File: Documents/tareasPOO/Homework_1/hw_8_py/hw8_jav/src/Libro.java
import java.util.Random; 

public class Libro {
    Random ran = new Random();
    int __id = 0;
    String __nombre = "";
    String __autor = "";
    boolean __es_rentado = false;

    public Libro(String nombre, String autor) {
        this.__id = ran.nextInt(0, 1000);
        this.__nombre = nombre;
        this.__autor = autor;
    }

    public String get_nombre() {
        return this.__nombre;
    }

    public String get_autor() {
        return this.__autor;
    }

    public int get_id() {
        return this.__id;
    }

    public void get_data() {
        System.out.println("El libro es "+ this.__nombre+" y su autor es "+this.__autor);
        System.out.println();
    }

    public boolean get_rentado() {
        return this.__es_rentado;
    }

    public void rentar() {
        this.__es_rentado = true;
    }

    public void devolver() {
        this.__es_rentado = false;
    }

}

File: Documents/tareasPOO/Homework_1/hw_8/Libro.py
import random

class Libro:
    __id = 0
    __nombre = ""
    __autor = ""
    __es_rentado = False

    def __init__(self, nombre, autor):
        self.__id = random.randint(1, 1000)
        self.__nombre = nombre
        self.__autor = autor
        
    def get_nombre(self):
        return self.__nombre
    
    def get_autor(self):
        return self.__autor
    
    def get_id(self):
        return self.__id
    
    def get_data(self):
        print(f"El libro es {self.__nombre} y su autor es {self.__autor}")
        
    def get_rentado(self):
        return self.__es_rentado
        
    def rentar(self):
        self.__es_rentado=True
        
    def devolver(self):
        self.__es_rentado=False